tests/source/issue-5525.rs

```diff
@@ -0,0 +1,9 @@
+// rustfmt-version: Two
+
+pub struct SomeCallback(
+    pub extern "C" fn(
+        long_argument_name_to_avoid_wrap: u32,
+        second_long_argument_name: u32,
+        third_long_argument_name: u32,
+    ),
+);
```

Comment:
If we're just testing that the input is idempotent then we don't need the `source/` test file.

Reply:
True, mb. I'll fix this now.